.bazelci/presubmit.yml

```diff
@@ -18,6 +18,7 @@ tasks:
     shell_commands:
     - "sudo apt -y update && sudo apt -y install automake autotools-dev cmake libtool m4 ninja-build"
     - "wget https://apt.llvm.org/llvm.sh && sudo bash llvm.sh 14"
+    - "sudo apt-get -qq install -y libclang-rt-14-dev"
     - "bazel/setup_clang.sh /usr/lib/llvm-14"
```

Comment:
i think this needs to be after the setup_clang line because it doesn't have the ppa yet, also can we add a TODO for removal? the script handles this case once we're on 15+ (alternatively i did https://github.com/envoyproxy/envoy/pull/26919 before i saw this)